packages/@angular/cli/models/config/config.ts

```diff
@@ -61,6 +61,12 @@ export class CliConfig<JsonType> {
     this._config.$$set(jsonPath, value);
   }
 
+  public getPaths(baseJsonPath: string, keys: string[]) {
+    const ret: { [k: string]: any } = {};
+    keys.forEach(key => ret[key] = this.get(`${baseJsonPath}.${key}`));
```

Comment:
Your code and his are not equivalent :)  
```typescript
return keys.reduce((acc, k) => {
  acc[key] = this.get(`${baseJsonPath}.${key}`);
  return acc;
}, {});
```
might be more correct but more verbose.